src/app/components/contacts/contacts.component.html

```diff
@@ -1,49 +1,116 @@
+      fill="#B49864"/>
+  </symbol>
+  <symbol id="doc" viewBox="0 0 24 24">
+    <path
+      d="M12.188 14.55c-.617 0-.977.588-.977 1.374 0 .79.37 1.35.983 1.35.617 0 .97-.588.97-1.374 0-.726-.347-1.35-.976-1.35z"
+      fill="#B49864"/>
+    <path
+      d="M14 2H6c-1.104 0-2 .896-2 2v16c0 1.104.896 2 2 2h12c1.104 0 2-.896 2-2V8l-6-6zM9.155 17.454c-.426.354-1.073.521-1.864.521-.475 0-.81-.03-1.038-.06v-3.971c.336-.054.773-.083 1.235-.083.768 0 1.266.138 1.655.432.42.312.684.81.684 1.522 0 .775-.282 1.309-.672 1.639zm2.99.546c-1.2 0-1.9-.906-1.9-2.058 0-1.211.772-2.116 1.966-2.116 1.241 0 1.92.929 1.92 2.045-.002 1.325-.806 2.129-1.986 2.129zm4.655-.762c.275 0 .581-.061.762-.132l.138.713c-.168.084-.546.174-1.037.174-1.397 0-2.117-.869-2.117-2.021 0-1.379.983-2.146 2.207-2.146.474 0 .833.096.995.18l-.186.726c-.187-.078-.444-.15-.768-.15-.726 0-1.29.438-1.29 1.338 0 .809.48 1.318 1.296 1.318zM14 9h-1V4l5 5h-4z"
+      fill="#B49864"/>
+    <path
+      d="M7.585 14.563c-.203 0-.335.018-.413.035v2.646c.078.018.204.018.317.018.828.005 1.367-.45 1.367-1.415.006-.84-.485-1.284-1.271-1.284z"
+      fill="#B49864"/>
+  </symbol>
+</svg>
+
 <div class="container">
-  <h2 class="page-title text-center" i18n="@@contactsTitle">Contacts</h2>
-  <div class="row">
-    <iframe class="map col-xs-12 col-sm-12 col-lg-6"
-            src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2594.251562988534!2d32.05918508347506!3d49.441962219007124!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x40d14b7e806fe79d%3A0x8962970b2d9c1d7d!2sCherkas%CA%B9kyy+Akademichnyy+Oblasnyy+Ukrayins%CA%B9kyy+Muzychno-Dramatychnyy+Teatr+Imeni+T.+H.+Shevchenka!5e0!3m2!1sru!2sua!4v1553702042151"
-            frameborder="0" style="border:0" allowfullscreen></iframe>
-    <ul class="col-xs-12 col-sm-12 col-lg-6 contacts-items-wrapper">
-      <li>
-        <div class="contact-item-name" i18n="@@contactsDirector">Director, Secretary</div>
-        <svg width="18" height="18" class="contact-item-pic u-svg-icon">
-          <use xlink:href="assets/images/sprite.svg#phone"></use>
-        </svg>
-        <div class="contact-item-content"><a href="tel:+380472360720">+38(0472)36-07-20</a>
-        </div>
-      </li>
-      <li>
-        <div class="contact-item-name" i18n="@@contactsAdmin">Administrator</div>
-        <svg width="18" height="18" class="contact-item-pic u-svg-icon">
-          <use xlink:href="assets/images/sprite.svg#phone"></use>
+  <h1 class="page_title" i18n="@@contactsTitle">Contacts</h1>
+
+  <div class="main">
+    <div class="info_wrapper">
+      <div class="info_item">
+        <svg class="icon">
```

Comment:
it's nice to always add width and height for this kind of icons - this way, in case smth will go wrong with css, your icons still would look ok

Reply:
Done